.dockerignore

```diff
@@ -1,3 +1,5 @@
+# SPDX-FileCopyrightText: Helmholtz-Zentrum Dresden-Rossendorf e.V. <https://www.hzdr.de>
+# SPDX-FileContributor: Benjamin Worpitz <benjaminworpitz@gmail.com>
```

Comment:
Benjamin contributed under the affiliation of HZDR and TUD, later partly as a pure open-source developer or LogMeIn employee, I do not know exactly.

Reply:
I only contributed for TUD, HZDR and as open-source developer. I did not contribute in the name of any other companies ;-)

Reply:
> Were HZDR and TU Dresden in parallel?
Initially only TUD. Later during my master-thesis also HZDR. Not sure how to attribute the external master-thesis though...

> Also do you remember when you left HZDR?
Officially this should have been 2015-09-31